hm_pyhelper/hardware_definitions.py

```diff
@@ -357,6 +357,29 @@ def is_raspberry_pi():
         'CONTAINS_IC_IDS': []
         },
 
+    # Smart Mimic / Mimiq Finestra
+    'COMP-FINESTRA': {
+        'FRIENDLY': 'Finestra Miner',
+        'SPIBUS': 'spidev0.0',
+        'KEY_STORAGE_BUS': '/dev/i2c-1',
+        'RESET': 17,
+        'MAC': 'wlan0',
+        'STATUS_LED': {
+            'TYPE': 'RGB',
+            'GPIO_NUMBERS_RGB': [20, 26, 7],
+            'GPIO_NUMBER_SINGLE': 20
+         },
+        'STATUS': 20,
```

Comment:
Overall looks good. Just one clarification needed. We are going to remove the `STATUS` field once all code using it is updated to use the `STATUS_LED` dict type field?

Reply:
Yes correct we will. I'll add that as a comment